csharp/test/Ice/udp/TestIntfI.cs

```diff
@@ -29,7 +29,7 @@ public void SendByteSeq(byte[] seq, IPingReplyPrx? reply, Current current, Cance
         {
             try
             {
-                reply!.Reply(cancel: cancel);
+                reply?.Clone(preferNonSecure: true).Reply(cancel: cancel);
```

Comment:
Should probably fix the Slice to disallow null reply.

Reply:
In one part of the test we actually pass `null`!